src/main/java/seedu/address/model/displayable/buyer/Buyer.java

```diff
@@ -47,19 +48,19 @@ public Buyer(Name name, Phone phone, Email email, Address address,
      * @param phone Phone number of the buyer.
      * @param email Email of the buyer.
      * @param address Home address of the buyer.
-     * @param buyHouseInfo Information on the house that the buyer is interested in.
+     * @param houseInfo Information on the house that the buyer is interested in.
      * @param tags Optional tags.
      * @param priority Priority level of the buyer.
      */
     public Buyer(Name name, Phone phone, Email email, Address address,
-                 BuyHouseInfo buyHouseInfo, Set<Tag> tags, Priority priority) {
+                 HouseInfo houseInfo, Set<Tag> tags, Priority priority) {
         super(name, phone, email, address, tags, priority);
-        requireAllNonNull(buyHouseInfo);
-        this.buyHouseInfo = buyHouseInfo;
+        requireAllNonNull(houseInfo);
+        this.houseInfo = houseInfo;
     }
 
-    public BuyHouseInfo getBuyHouseInfo() {
-        return buyHouseInfo;
+    public HouseInfo getInfo() {
```

Comment:
Could you refactor this to getHouseInfo() as well? (following from the refactoring of Info to HouseInfo)